changes/unreleased/Added-20240808-140515.yaml

```diff
@@ -0,0 +1,5 @@
+kind: Added
```

Comment:
This is not a highlight

Reply:
Should I remove this changie if it's not important/hightlight?